src/dug/core/annotate.py

```diff
@@ -148,11 +148,7 @@ def expand_identifier(self, identifier, query_factory, kg_filename):
                 data=query).json()
 
             # Case: Skip if empty KG
-            logger.debug(response)
-            if 'knowledge_graph' not in response:
-                logger.debug(f"Did not find a knowledge graph for {query}")
-                return []
-            if not len(response['knowledge_graph']['nodes']):
+            if not len(response["message"]['knowledge_graph']['nodes']):
```

Comment:
Accessing nested dicts should have error handling in case one of the keys is missing

Reply:
modified to 
```
            if not len(response.get("message", {}).get("knowledge_graph", {}).get("nodes", [])):
                logger.debug(f"Did not find a knowledge graph for {query}")
                logger.debug(f"{self.url} returned response: {response}")

Reply:
👍🏻  although maybe 
```
try:
    nodes = response["message"]['knowledge_graph']['nodes']
except KeyError:
    ....
```

Reply:
updated it to 
```            
          try:
                if not len(response["message"]["knowledge_graph"]["nodes"]):
                    logger.debug(f"Did not find a knowledge graph for {query}")
                    logger.debug(f"{self.url} returned response: {response}")
                    return []
            except KeyError as e:
                logger.error(f"Could not find key: {e} in response: {response}")

Reply:
👍🏻  much cleaner